### PR TITLE
Add userguide information to docs

### LINKED ISSUE #2735: docs should link to User Guide and other resources
## Expected behavior ##
The docs https://www.mdanalysis.org/docs/ make users aware of other important resources such as the Quick Start Guide and the whole User Guide (perhaps as well the Learning MDAnalysis overview page).

There are numerous other issues with the docs (which need major cleanup) but this is a simple one that can be easily fixed.

## Actual behavior ##

No links.

## Code to reproduce the behavior ##

Read https://www.mdanalysis.org/docs/

## Current version of MDAnalysis ##

- Which version are you using? (run `python -c "import MDAnalysis as mda; print(mda.__version__)"`) 1.0.0

### PR DESCRIPTION
**Pull request by IAlibay**
Fixes #2735

Changes made in this Pull Request:
 - Adds extra information on the userguide to the main docs page.

PR Checklist
------------
 - Tests?
 - [x] Docs?
 - CHANGELOG updated?
 - [x] Issue raised/referenced?

### CODE REVIEW

**Comment by IAlibay** [comment]
@MDAnalysis/coredevs this should be the last PR on the list of things for 2.0. Should be a short one if anyone has a few minutes.

File: package/doc/sphinx/source/index.rst [hunk 1]
```diff
@@ -59,6 +59,19 @@ members agree and adhere to --- please read it.
 
 .. _installation-instructions: 
 
+
+User Guide
+==========
+The MDAnalysis `userguide`_ provides comprehensive information on how to
```
Comment (orbeckst) [inline package/doc/sphinx/source/index.rst:65]: I'd write it *User Guide* consistently, at least that's how https://userguide.mdanalysis.org/stable/index.html refers to itself.
```suggestion
The MDAnalysis `User Guide`_ provides comprehensive information on how to
```

File: package/doc/sphinx/source/index.rst [hunk 2]
```diff
@@ -59,6 +59,19 @@ members agree and adhere to --- please read it.
...
+use the library. We would recommend that new users have a look at the
+`quick start guide`_. The userguide also has a set of `examples`_ on how to
+use the MDAnalysis library which may be of interest.
+
+.. _`userguide`: https://userguide.mdanalysis.org/stable/index.html
```
Comment (orbeckst) [inline package/doc/sphinx/source/index.rst:70]:
```suggestion
.. _`User Guide`: https://userguide.mdanalysis.org/stable/index.html
```

File: package/doc/sphinx/source/index.rst [hunk 3]
```diff
@@ -59,6 +59,19 @@ members agree and adhere to --- please read it.
 
 .. _installation-instructions:
```
Comment (orbeckst) [inline package/doc/sphinx/source/index.rst:60]: Put this link target in front of installation.
```suggestion
```

File: package/doc/sphinx/source/index.rst [hunk 4]
```diff
@@ -59,6 +59,19 @@ members agree and adhere to --- please read it.
...
+The MDAnalysis `userguide`_ provides comprehensive information on how to
+use the library. We would recommend that new users have a look at the
+`quick start guide`_. The userguide also has a set of `examples`_ on how to
+use the MDAnalysis library which may be of interest.
+
+.. _`userguide`: https://userguide.mdanalysis.org/stable/index.html
+.. _`quick start guide`: https://userguide.mdanalysis.org/stable/examples/quickstart.html
+.. _`examples`: https://userguide.mdanalysis.org/stable/examples/README.html
+
+
```
Comment (orbeckst) [inline package/doc/sphinx/source/index.rst:74]: Moved link target
```suggestion

.. _installation-instructions: 
```

File: package/doc/sphinx/source/index.rst [hunk 5]
```diff
@@ -127,15 +140,20 @@ Source Code
...
 https://github.com/MDAnalysis/mdanalysis/ under the `GNU Public
-Licence, version 2`_. Obtain the sources with `git`_
+Licence, version 2`_. Obtain the sources with `git`_.
 
 .. code-block:: bash
 
    git clone https://github.com/MDAnalysis/mdanalysis.git
 
+
+The userguide provides more information on how to
```
Comment (orbeckst) [inline package/doc/sphinx/source/index.rst:150]:
```suggestion
The `User Guide`_ provides more information on how to
```

**Review by orbeckst — APPROVED** [review]
Looks good, I'll just apply my suggestions.

**Comment by IAlibay** [comment]
Thanks @orbeckst !

**Comment by orbeckst** [comment]
Some minor fiddling with the text... hopefully it still builds correctly.

**Comment by orbeckst** [comment]
Glad that you hadn't merged it yet. Just squash it all together if you're happy with it.

**Comment by orbeckst** [comment]
@lilyminium anything you want to add here?

**Review by lilyminium — APPROVED** [review]
Thank you @IAlibay!

**Pull request merged**